extension/src/fileSystem/dvcYaml/index.ts

```diff
@@ -0,0 +1,55 @@
+import { join } from 'path'
```

Comment:
Didn't use this either, will remove it too

Reply:
We would have only wanted one instance of `extension/src/fileSystem/data/index.ts` per repository anyway.